app/controllers/api/stages_controller.rb

```diff
@@ -19,10 +18,25 @@ def clone
     end
   end
 
+  def update
+    stage.deploy_groups = deploy_groups
```

Comment:
... good enough for now, but I'd expect someone wants to do more then deploy_groups :D